src/components/statevector_component/aggregation.py

```diff
@@ -343,7 +460,15 @@ def force_native_res_pixels(config, clusters, sensitivities):
         cluster_pairs    [(tuple)]: cluster pairings
     Returns:             [double] : updated sensitivities
     """
+    plumes = SRON_plumes(config)
```

Comment:
This should be an option rather than happening automatically. Maybe we need another config variable eg. `PointSourceDatasets: ["SRON"]`. Making it a list allows for easier extension to other public datasources in the future eg. `IMEO`